DragAndDrop/src/androidTest/java/com/microsoft/device/display/samples/draganddrop/DragAndDropLandscapeTest.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.draganddrop
+
+import androidx.test.filters.LargeTest
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.microsoft.device.display.samples.draganddrop.utils.setOrientationLeft
+import com.microsoft.device.display.samples.draganddrop.utils.setOrientationRight
+import com.microsoft.device.display.samples.draganddrop.utils.unfreezeRotation
+import org.junit.After
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@LargeTest
+class DragAndDropLandscapeTest : DragAndDropPortraitTest() {
```

Comment:
If you have a test class which extends another test class, when you run all the UI tests, the ones from DragAndDropPortraitTest will be run twice, maybe we could find a way to prevent this?

Reply:
Done